runtime/jvmti/jvmtiHelpers.c

```diff
@@ -1929,3 +1929,20 @@ genericWalkStackFramesHelper(J9VMThread *currentThread, J9VMThread *targetThread
 
 	return rc;
 }
+
+#if JAVA_SPEC_VERSION >= 19
+J9VMContinuation *
+getJ9VMContinuationToWalk(J9VMThread *currentThread, J9VMThread *targetThread, j9object_t threadObject)
+{
+	J9VMContinuation *continuation = NULL;
+	if (IS_VIRTUAL_THREAD(currentThread, threadObject)) {
+		if (NULL == targetThread) {
```

Comment:
Based on the discussion, this will always be NULL, right?

Reply:
Or does getVMThread return the carrier vmThread for mounted?

Reply:
Yes, it returns the J9VMThread of the carrier thread when a vthread is mounted.